parlai/crowdsourcing/tasks/turn_annotations_static/run_task.py

```diff
@@ -82,6 +83,32 @@ def setup_mephisto(launch_config):
         f'--onboarding-qualification turn-ann-s-onb '
         f"-use-onboarding True "
     )
+    # Optional flags:
+    try:
```

Comment:
This exception handling will not be needed with Hydra because it'll just use the default value of the param, but this is fine for now

Reply:
Agree